app/javascript/components/Home/Service/WorthPhotoContainer.1.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from "react";
+import { FadeInLeft } from "../../Constants/Constants";
+import {
+  postFetch,
+  deleteFetch,
+  getFetch
+} from "../../Constants/FetchComponent";
+import { DeleteButton, AddButton } from "../../Constants/Buttons";
+
+class WorthPhotoContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      url: "worth_photos",
+      photoData: [],
+      photo: "",
+      refreshKey: false
+    };
+  }
+
+  toggleRefreshKey = () => this.setState({ refreshKey: true });
+
+  onChange = e => this.setState({ [e.target.name]: e.target.value });
+
+  onSubmit = event => {
```

Comment:
You can use async/await notation here instead of using .then() and .catch()

example:
```
async onSubmit = event => {
      stuff
      more stuff
      await postFetch()
      this.toggleRefresh
}